tests_integration/__tests__/__snapshots__/arg-parsing.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`boolean flags do not swallow the next argument (stdout) 1`] = `
 exports[`boolean flags do not swallow the next argument (write) 1`] = `Array []`;
 
 exports[`deprecated option values are warned (stderr) 1`] = `
-"[warn] \`--trailing-comma\` without an argument is deprecated. Prettier now treats it as \`--trailing-comma=es5\`.
+"[warn] --trailing-comma without an argument is deprecated, now treats it as --trailing-comma=es5.
```

Comment:
"now treats it" without a subject isn't valid; maybe we should say "this is treated as"? It's passive voice but... I think it's clearer

Reply:
How about `we now treat it as ...`?

Reply:
perfect :+1:

Reply:
May I do a drive-by interjection about comma splices? That should be "... deprecated; we now treat it as", rather than separated by a semicolon.